source/ui/win32/Win32DisplayWindow.ooc

```diff
@@ -15,7 +15,7 @@ version(windows) {
 Win32DisplayWindow: class extends DisplayWindow {
 	_backend: Win32Window
 	init: func (size: IntVector2D, title: String) {
-		super()
+		super(size, title)
```

Comment:
Interesting. `DisplayWindow` has `init: func (size: IntVector2D, title: String)` which does... nothing. I guess we should rather change its constructor to just be `init: func`. @sebastianbaginski ?